src/tutorial/meshvis.md

```diff
@@ -127,8 +135,8 @@ MFEM can import meshes saved in Exodus II format generated with  <a href="https:
 
 ---
 
-### <i class="fa fa-check-square-o"></i>&nbsp; Visualizing results in VisIt and Paraview
-To save the simulation results from example 1 (ex1p.cpp) in ParaView format, add the following lines just before step 17 in the file.
+### <i class="fa fa-check-square-o"></i>&nbsp; Visualizing results in VisIt and ParaView
+To save the simulation results from the parallel version of Example 1 ([ex1p.cpp](https://github.com/mfem/mfem/blob/master/examples/ex1p.cpp)) in ParaView format, add the following lines just before step 17 in the file.
```

Comment:
Should the user see the "Remember to compile each numbered example before executing its sample runs" reminder here, or are they only saving data (not visualizing it)?